UnityProject/Assets/Scripts/Objects/Doors/DoorMasterController.cs

```diff
@@ -174,7 +174,14 @@ private void Awake()
 				openSortingLayer = SortingLayer.NameToID("Machines");
 			}
 
-			doorAnimator.SyncDoorStatus(doorAnimator.SyncDoorUpdateType, DoorAnimatorV2.DoorUpdateType.Close);
+			if (IsClosed)
```

Comment:
This if else could be compressed to one line, or at the least 2 lines. Not a big deal, up to you which you prefer